app/assets/javascripts/map.js

```diff
@@ -0,0 +1,49 @@
+$(function() {
+  var courtsRequest;
+  var map;
+
+  var createMap = function(latitude, longitude){
+    L.mapbox.accessToken = "pk.eyJ1IjoiZWxpa2ltIiwiYSI6Ik1YSm9TY3cifQ.-aONewTGq_GmSuvGS7xQaQ";
```

Comment:
This key should be extracted to an environment variable and kept out of the repository.